storage/acl.go

```diff
@@ -361,6 +399,16 @@ func toBucketProjectTeam(p *raw.BucketAccessControlProjectTeam) *ProjectTeam {
 	}
 }
 
+func projectTeamFromProto(p *storagepb.ProjectTeam) *ProjectTeam {
```

Comment:
This functions seems to be the same as the func `fromProtoToObjectProjectTeam ` on L422?

Reply:
You're right! Getting rid of `fromProtoToObjectProjectTeam` in favor of the generic method I've added.